screen/wallets/entropyGenerator.js

```diff
@@ -0,0 +1,426 @@
+    return generateEntropy(normalizedPoints);
+  };
+
+  const createWallet = async () => {
+    setIsLoading(true);
+
+    let w;
+
+    if (selectedWalletType === ButtonSelected.OFFCHAIN) {
+      createLightningWallet(w);
+    } else if (selectedWalletType === ButtonSelected.ONCHAIN) {
+      const entropy = await generateEntropyDrawing();
+      const shuffledEntropy = getNRandomElementsFromArray(entropy, 16);
+      if (entropy.length < 192) {
+        alert('Your entropy is too small. Please draw more lines.');
+        setIsLoading(false);
+        return;
+      }
+      if (selectedIndex === 2) {
+        // zero index radio - HD segwit
+        w = new HDSegwitP2SHWallet();
+        w.setLabel(label || loc.wallets.details_title);
+      } else if (selectedIndex === 1) {
+        // btc was selected
+        // index 1 radio - segwit single address
+        w = new SegwitP2SHWallet();
+        w.setLabel(label || loc.wallets.details_title);
+      } else {
+        // btc was selected
+        // index 2 radio - hd bip84
+        if (isTestMode) {
+          w = new HDSegwitBech32Wallet({ network: bitcoin.networks.testnet });
+        } else {
+          w = new HDSegwitBech32Wallet();
+        }
+        w.setLabel(label || loc.wallets.details_title);
+      }
+      if (selectedWalletType === ButtonSelected.ONCHAIN) {
+        await w.generateMnemonicFromEntropy(shuffledEntropy);
+        addWallet(w);
+        await saveToDisk();
+        A(A.ENUM.CREATED_WALLET);
+        ReactNativeHapticFeedback.trigger('notificationSuccess', { ignoreAndroidSystemSettings: false });
+        if (w.type === HDSegwitP2SHWallet.type || w.type === HDSegwitBech32Wallet.type) {
+          navigate('PleaseBackup', {
+            walletID: w.getID(),
+          });
+        } else {
+          goBack();
+        }
+      }
+    } else if (selectedWalletType === ButtonSelected.VAULT) {
+      setIsLoading(false);
+      navigate('WalletsAddMultisig', { walletLabel: label.trim().length > 0 ? label : loc.multisig.default_label });
+    } else if (selectedWalletType === ButtonSelected.LDK) {
+      setIsLoading(false);
+      createLightningLdkWallet(w);
+    }
+  };
+
+  const createLightningLdkWallet = async (wallet) => {
+    const foundLdk = wallets.find((w) => w.type === LightningLdkWallet.type);
+    if (foundLdk) {
+      return alert('LDK wallet already exists');
+    }
+    setIsLoading(true);
+    wallet = new LightningLdkWallet();
+    wallet.setLabel(label || loc.wallets.details_title);
+
+    await wallet.generate();
+    await wallet.init();
+    setIsLoading(false);
+    addWallet(wallet);
+    await saveToDisk();
+
+    A(A.ENUM.CREATED_WALLET);
+    ReactNativeHapticFeedback.trigger('notificationSuccess', { ignoreAndroidSystemSettings: false });
+    navigate('PleaseBackupLdk', {
+      walletID: wallet.getID(),
+    });
+  };
+
+  const createLightningWallet = async (wallet) => {
```

Comment:
Can some of this be scrapped?

Reply:
@muursh  yes this was the old code , but here in this case I can remove completely  **Lightening Network** from **Entropy Generator Screen File**  .